sdk/keyvault/azure-keyvault-keys/tests/test_crypto_client.py

```diff
@@ -276,6 +276,23 @@ def test_encrypt_local(self, azure_keyvault_url, **kwargs):
             result = crypto_client.decrypt(result.algorithm, result.ciphertext)
             self.assertEqual(result.plaintext, self.plaintext)
 
+    @KeyVaultPreparer()
+    def test_encrypt_local_from_jwk(self, azure_keyvault_url, **kwargs):
```

Comment:
It's also possible to test private operations locally, using an imported key for public operations (the neighboring `keys.py` has suitable keys). Doesn't need to be in this PR but it's a test gap to fill before the next stable release. Also, need to cover EC keys as well.